appender_test.go

```diff
@@ -1647,6 +1649,51 @@ func TestAppenderTracing(t *testing.T) {
 	testAppenderTracing(t, 400, "failure")
 }
 
+func TestAppenderOtelTracing(t *testing.T) {
```

Comment:
This test is currently failing, as reported spans & observed logs is `0`.

Reply:
Test now fixed, I was shutting down the exporter, but being it in memory this was discarding the spans.